RadarSDK/Radar.m

```diff
@@ -1232,11 +1232,7 @@ + (void)flushLogs {
 
     [[RadarAPIClient sharedInstance] syncLogs:flushableLogs
                             completionHandler:^(RadarStatus status) {
-                                if (onComplete) {
-                                    [RadarUtils runOnMainThread:^{
-                                        onComplete(status);
-                                    }];
-                                }
+                               onComplete(status); 
```

Comment:
Do we know what thread is being used to call this completion handler (could also be the main thread)

Reply:
It's not easy to reason through which thread will call this chunk at runtime. It won't t hurt to be explicit in this case. I should just send it to a another thread.